gnome-initial-setup/eos-demo-mode-force-gdm-restart

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+systemctl reload gdm.service
```

Comment:
I think that instead of a separate script for this you could use systemd's dbus API; there's a `ReloadUnit()` method on the `org.freedesktop.systemd1.Manager` interface exposed by `/org/freedesktop/systemd1` at `org.freedesktop.systemd1`